src-vue/src/views/ThunderstoreModsView.vue

```diff
@@ -21,7 +29,7 @@
                 </div>
 
                 <!-- Mod cards -->
-                <thunderstore-mod-card v-for="mod of modsList" v-bind:key="mod.name" :mod="mod" />
+                <thunderstore-mod-card v-for="mod of currentPageMods" v-bind:key="mod.name" :mod="mod" />
```

Comment:
It might make sense to display a "_Next Page_" button at the bottom of the list, so that one doesn't have to scroll back up to go to the next page ^^

Reply:
Addressed in 6689e164a9ac139cfe8c925792db13451348af10
(I added another pagination component on view bottom)